docker-compose.prod.yml

```diff
@@ -16,3 +19,40 @@ services:
 
   api:
     restart: always
+
+  traefik:
+      container_name: traefik
+      image: traefik:v2.0
+      command:
+          - "--api.dashboard=true"
+          - "--providers.docker=true"
+          - "--entryPoints.web.address=:80"
+          - "--entryPoints.websecure.address=:443"
+          - "--certificatesResolvers.le.acme.email=${TRAEFIK_EMAIL}"
+          - "--certificatesResolvers.le.acme.storage=acme.json"
+          - "--certificatesResolvers.le.acme.tlsChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge.entryPoint=web"
+          - "--log.level=INFO"
+      restart: always
+      ports:
+          - 80:80
+          - 443:443
+          - 8080:8080
+      networks:
+          - proxy
+      volumes:
+          - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Can this one be read only?

Reply:
I don't know. I found this but haven't tested it. https://chriswiegman.com/2019/11/protecting-your-docker-socket-with-traefik-2/